updater/lib/dependabot/updater/operations/refresh_security_update_pull_request.rb

```diff
@@ -197,6 +213,24 @@ def check_and_update_pull_request(dependencies)
         # rubocop:enable Metrics/MethodLength
         # rubocop:enable Metrics/CyclomaticComplexity
 
+        # Feature testing to fix issues related with scenarios when a PR is updated while containing
+        # the same version in existing PR and dependeabot updater closes and raised same PR
+        sig { params(lead_dep_name: String, existing_pr_lead_dep_ver: String).returns(T::Boolean) }
+        def pr_lead_dep_latest_ver(lead_dep_name, existing_pr_lead_dep_ver)
+          job.existing_pull_requests.each do |existing_pr|
+            existing_pr.dependencies.each do |deps|
+              next unless (deps.name.eql? lead_dep_name) && (deps.version.eql? existing_pr_lead_dep_ver)
```

Comment:
you may want to lower-case the dependency name here for comparison

Reply:
this code is no longer relevant